src/dashboard/Receiving/Receiving.js

```diff
@@ -0,0 +1,158 @@
+        <div className={classes.headerContainer}>
+          <Icon name="cached" className={classes.cachedIcon} />
+          <Typography variant="h4">Self-Tribute</Typography>
+        </div>
+        <Container className={classes.contentContainer}>
+          <Paper elevation={5} className={classes.unclaimedTributeContainer}>
+            <Typography variant="body1">
+              <b>450</b>{' '}
+              <Icon name="baseCurrency" className={classes.baseCurrencyIcon} />{' '}
+              from your principal are generating interest for you.
+            </Typography>
+          </Paper>
+          <Divider className={classes.divider} />
+        </Container>
+      </Container>
+    );
+  };
+
+  const getActiveInflows = () => {
+    return (
+      <Container className={classes.container}>
+        <div className={classes.headerContainer}>
+          <Icon name="faucetOn" className={classes.headerImage} />
+          <Typography variant="h4">Active Tributes</Typography>
+        </div>
+        <Container className={classes.contentContainer}>
+          <CustomTable
+            headings={[
+              'Sender',
+              'Asset',
+              'Current APR',
+              'Tribute Amount',
+              'Flowing Since',
+              'Total Received'
+            ]}
+            rows={[[1, 2, 3, 4, 5, 6], [1, 2, 3, 4, 5, 6], [1, 2, 3, 4, 5, 6]]}
+          />
+          <Paper elevation={5} className={classes.unclaimedTributeContainer}>
+            <Typography variant="body1">
+              Ready to redeem: <b>344</b>{' '}
+              <Icon name="baseCurrency" className={classes.baseCurrencyIcon} />
+            </Typography>
+            <div>
+              <Button
+                onClick={redeemTribute()}
+                variant="contained"
+                color="primary"
+                className={classes.redeemButton}
+              >
+                <Icon name="receiveMoney" className={classes.buttonIcon} />
+                Redeem
+              </Button>
+            </div>
+          </Paper>
+          <Divider className={classes.divider} />
+        </Container>
+      </Container>
+    );
+  };
+
+  const getInactiveInflows = () => {
+    return (
+      <Container className={classes.container}>
+        <div className={classes.headerContainer}>
+          <Icon name="faucetOff" className={classes.headerImage} />
+          <Typography variant="h4">Inactive Tributes</Typography>
+        </div>
+        <Container className={classes.contentContainer}>
+          <CustomTable
+            headings={[
+              'Sender',
+              'Asset',
+              'Average APR',
+              'Tribute Amount',
+              'Tribute Length',
+              'Total Received'
+            ]}
+            rows={[[1, 2, 3, 4, 5, 6], [1, 2, 3, 4, 5, 6], [1, 2, 3, 4, 5, 6]]}
+          />
+        </Container>
+      </Container>
+    );
+  };
+
+  return (
+    <div>
+      {getSelfTribute()}
```

Comment:
why do function calls instead of react tags?

Reply:
like <SelfTribue/>?
no reason.

Reply:
can we make these tags instead?

Reply:
I think this is ok for now